great-idea/css/index.css

```diff
@@ -63,4 +63,333 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* START OF CUSTOM CSS */
+
+body {
+	width: 900px;
+	margin: auto;
+	text-align: center;
+	background-color: rgb(14, 14, 20);
+	color: rgb(201, 200, 200);
+}
+
+p {
+	margin: 20px 20px 0 0;
+}
+
+hr {
+	border: 1px solid white;
+}
+
+h3 {
+	font-family: 'Open-Sans', sans-serif;
+	font-weight: 600;
+	color: white;
+}
+
+h1 {
+	color: white;
+}
+
+/* Nav Section */
+
+nav {
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	position: fixed;
+	top: 0;
+	padding: 10px 0;
+	text-align: left;
+	background-image: linear-gradient(to right, rgb(38, 38, 68) , rgb(90, 90, 161));
+	z-index: 1;
+	width: 900px;
+}
+
+nav a.nav-link {
+	text-decoration: none;
+	color: white;
+	transition: 1s;
+}
+
+nav a:hover {
+	font-size: 120%;
+}
+
+nav img {
+	margin-left: 54px;
+}
+
+
+/* Header Section */
+
+	/* home page header css */
+.home-header {
+	background: url("/great-idea/img/header-img.png") no-repeat center center;
+	margin-top: 30px;
+	background-size: 55%;
+	z-index: -1;
+	padding: 150px;
+}
+
+.main-header {
+	display: flex;
+	flex-direction: column;
+	width: 70%;
+	justify-content: center;
+	margin: auto;
+}
+
+.home-header h1 {
+	font-size: 60px;
+	font-family: 'Open-Sans', sans-serif;
+	text-align: center;
+	color: rgb(2, 30, 56);
+	background-color: rgba(255, 255, 255, 0.8);
+	border-radius: 10px;
+	padding: 20px;
+}
+
+.main-header button {
+	margin: auto;
+	background-color: rgb(255, 255, 255);
+	border: 1px solid rgb(97, 97, 97);
+	color: rgb(27, 27, 27);
+	padding: 7px 40px;
+	text-align: center;
+	text-decoration: none;
+	font-size: 16px;
+	width: 50%;
+	transition: 2s;
+	cursor: pointer;
```

Comment:
Some people hate pointers on buttons and there is a big debate on the affordance of a button and such, but I'm glad you put this here, I personally find it to be a nice UI touch to be honest.  This, overall is a well designed button.